resources/views/backups/modal.blade.php

```diff
@@ -21,7 +21,7 @@
         </button>
         <p class="help-block">
             A <strong>Database and Resources</strong> backup includes any files that aren't version controlled.
-            At the moment this includes: member profile pictures, resources and election manifestos.
+            At the moment this includes: member profile pictures, resources, breakage report images and election manifestos.
```

Comment:
Is this correct? It probably should, but you'll need to configure it: https://github.com/backstage-technical-services/website/blob/main/config/backup.php#L29